src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Plot XY trajectory from inspiral run, concatenating all segments
+
+import glob
+import os
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+@click.command()
+@click.argument("simulation_name", type=str)
```

Comment:
Replace this option with:

```py
@click.argument(
    "h5_files",
    nargs=-1,
    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
)
```

This means the user won't pass a simulation directory to the CLI, but they will pass the reduction files directly as a glob pattern like this:

```
spectre plot trajectories path/to/Segment_*/BbhReductions.h5
```

This is more flexible because no assumptions are made on the directory structure and the user can choose which segments/files to plot. It's listed in the guidelines here: https://spectre-code.org/spectre_writing_python_bindings.html#autotoc_md155

Then you don't need the `select_reduction_files` function anymore.